recipe/meta.yaml

```diff
@@ -12,8 +12,9 @@ build:
   number: 1002
   skip: True  # [py<34]
   script:
-    - "{{ PYTHON }} -m pip install . --no-deps -vv"
-    - python -m ipykernel install --prefix {{ PREFIX }}
+    - "{{ PYTHON }} setup.py bdist_wheel"
```

Comment:
Actually, I forgot about this and we should probably go back to the old way in the next release.